CMSgov/design-system
# [WNMGDS-327] Change radio and checkbox checkmark to be created with SCSS instead of image

This fixes #568

- Updated the checkmark to be created with SCSS, similar to how Gov.uk does it. https://design-system.service.gov.uk/components/checkboxes/
- Removed the white-checkmark.svg as it was only used i the choice component 
- Updated the SCSS and organized it a bit better

**To test** 

- Open up the doc site and visit the choice page
- See that the checkmark is now visible on checked or selected items for both radio buttons and checkboxes


The screenshot from my test (checks are visible)

![Screen Shot 2020-07-08 at 3 11 20 PM](https://user-images.githubusercontent.com/1449852/86960343-60cda380-c12d-11ea-9bf2-ba9d8b49f287.png)
